src/fortran/cantera_thermo.f90

```diff
@@ -545,6 +552,13 @@ subroutine ctthermo_getEnthalpies_RT(self, h_rt)
       self%err = th_getenthalpies_rt(self%thermo_id, h_rt)
     end subroutine ctthermo_getenthalpies_rt
 
+    subroutine ctthermo_getGibbs_RT(self, grt)
+      implicit none
+      type(phase_t), intent(inout) :: self
+      double precision, intent(out) :: grt(self%nsp)
+      self%err = th_getpartialmolarenthalpies(self%thermo_id, grt)
+    end subroutine ctthermo_getGibbs_RT
```

Comment:
Presumably this should call `th_getgibbs_rt` instead of `th_getpartialmolarenthalpies`.